projects/ngx-location-viewer/src/lib/components/ngx-location-viewer.component.scss

```diff
@@ -22,10 +24,17 @@
     .whatishere-type {
         width: 4.0625rem;
     }
-    
+
     .whatishere-image {
         overflow: hidden;
     }
+
+    @include to($screen-xs-max) {
+        .o-leaflet__content.has-content {
+            padding: 0.75rem;
+            width: calc(100% - 65px);
```

Comment:
I'm not sure why you need to do this, plus we always use rem values to ensure scalability.